doc/architecture/trace_file_binary_and_human_readable_formats.adoc

```diff
@@ -2,11 +2,16 @@ ifndef::include-only-once[]
 :root-path: ../
 include::{root-path}_config.adoc[]
 endif::[]
-= OSI trace file naming conventions
+= Simple binary and Human-readable Formats
 
-**Name format**
+== Binary .osi Format
+Messages are separated by a four-byte, little-endian, unsigned integer specifying the length of each message.
 
-The names of OSI trace files should have the following format:
+== Human-readable .txth Format
+Messages are stored as plain text, separated by newlines.
+
+== Naming Convention
```

Comment:
The naming convention should be recommended for all trace file formats (incl. mcap).

Reply:
For documentation purposes, the conclusion of the meeting was that further discussion is required. There are differing opinions on whether a naming convention for MCAP files should be recommended, mandatory or not be required at all. Some support this requirement (to ease the work with external partners), while others are concerned about the complications it may introduce, such as handling situations where trace files are created dynamically over and over again while working on other tasks (as trace files are merely a tool, not the end goal).

If a naming convention for MCAP files is included in the specification, several considerations must be addressed:

- An MCAP file can technically store messages (in different channels) from various OSI versions.
- Including the Protobuf version in the naming may not offer any substantial advantage.
- Should the "number of frames" reflect only OSI data, or should it also account for non-OSI data that might be present in the MCAP?
- How should the "type" be defined if multiple channels contain different types of top-level messages (a proposed solution is to use "multi"), or if there are multiple instances of the same kind of top-level message?
- What point in time should the "timestamp" represent? We agreed to wait for https://github.com/OpenSimulationInterface/open-simulation-interface/issues/834

Reply:
During today's meeting we proposed a convention (see comments below or updated files).  Thus resolving this thread.